doc/releases/changelog-0.11.0.md

```diff
@@ -23,175 +27,235 @@
+  its own analogous implementation of a transform available in PennyLane, the transform will be expanded 
+  according to rules provided by PennyLane.
+
+  For example, consider this workflow that contains two PennyLane transforms: `cancel_inverses` and 
+  `single_qubit_fusion`. Catalyst has its own implementation of `cancel_inverses` in the `passes` module, 
+  and will smartly invoke its implementation intead. Conversely, Catalyst does not have its own implementation
+  of `single_qubit_fusion`, and will therefore resort to PennyLane's implementation of the transform.
+
+  ```python
   dev = qml.device("lightning.qubit", wires=1)
 
   @qjit(experimental_capture=True)
-  def func(x: float):
+  def func(r1, r2):
+
+      @qml.transforms.single_qubit_fusion
       @qml.transforms.cancel_inverses
       @qml.qnode(dev)
-      def circuit(x: float):
-          qml.RX(x, wires=0)
+      def circuit(r1, r2):
+          qml.Rot(*r1, wires=0)
+          qml.Rot(*r2, wires=0)
+          qml.RZ(r1[0], wires=0)
+          qml.RZ(r2[0], wires=0) 
+
           qml.Hadamard(wires=0)
           qml.Hadamard(wires=0)
-          return qml.expval(qml.PauliZ(0))
+          
+          return qml.expval(qml.PauliZ(0))  
+
+      return circuit(r1, r2)
+  ```
 
-      return circuit(x)
+  ```pycon
+  >>> r1 = jnp.array([0.1, 0.2, 0.3])
+  >>> r2 = jnp.array([0.4, 0.5, 0.6])
+  >>> func(r1, r2)
+  Array(0.7872403, dtype=float64)
   ```
 
-* Changes to reduce compile time:
+<h3>Improvements 🛠</h3>
+
+* The `merge-rotations` peephole optimization pass has been extended to also merge compatible rotation 
+  gates (either both controlled, or both uncontrolled) where rotation angles are any combination of 
+  static constants or dynamic values.
+  [(#1489)](https://github.com/PennyLaneAI/catalyst/pull/1489)
 
-  - Turn off MLIR's verifier.
+* Several changes have been made to reduce compile time:
+  * MLIR's verifier has been turned off.
     [(#1513)](https://github.com/PennyLaneAI/catalyst/pull/1513)
-  - Remove unnecessary I/O.
+  * Unnecessary I/O has been removed.
     [(#1514)](https://github.com/PennyLaneAI/catalyst/pull/1514)
     [(#1602)](https://github.com/PennyLaneAI/catalyst/pull/1602)
-  - Sort improvements to reduce complexity and memory.
+  * Improvements have been made to reduce complexity and memory.
     [(#1524)](https://github.com/PennyLaneAI/catalyst/pull/1524)
-  - Lazy IR canonicalization and LLVMIR textual generation.
+  * IR canonicalization and LLVMIR textual generation is now performed lazily.
     [(#1530)](https://github.com/PennyLaneAI/catalyst/pull/1530)
-  - Speed up how tracers are overwritten for hybrid ops.
+  * Speed up how tracers are overwritten for hybrid ops.
     [(#1622)](https://github.com/PennyLaneAI/catalyst/pull/1622)
 
-* Catalyst now decomposes non-differentiable gates when in a gradient method.
+* Catalyst now decomposes non-differentiable gates when differentiating through workflows.
```

Comment:
We should also mention that verification has been improved for the parameter-shift method, circuits are now verified to be fully compatible with Catalyst's parameter-shift implementation before compilation.

Reply:
```suggestion
* Catalyst now decomposes non-differentiable gates when differentiating through workflows. Additionally, with `diff_method=parameter-shift`,
  circuits are now verified to be fully compatible with Catalyst's parameter-shift implementation before compilation.
```